### DISCUSSION TITLE
install error

### DISCUSSION THREAD

**Discussion by blockmood** [gatsbyjs/gatsby#31302] [Help]
<img width="730" alt="企业微信截图_87441c8f-ac5d-418e-8c54-643a4a4cca5e" src="https://user-images.githubusercontent.com/22879524/117390906-8a0fb980-af21-11eb-90a6-99c8570cada3.png">

## Description

Describe the issue that you're seeing.

### Steps to reproduce

Clear steps describing how to reproduce the issue. **Please** link to a reproduction, this makes your issue _much_ easier to diagnose (seriously).

How to Make a Minimal Reproduction: https://www.gatsbyjs.com/contributing/how-to-make-a-reproducible-test-case/

### Expected result

What should happen?

### Actual result

What happened.

### Environment

Run `gatsby info --clipboard` in your project directory and paste the output here. Also name any `flags` you use inside `gatsby-config.js`.

**Comment by LekoArts** [comment]
Please make sure that you have the XCode tools installed so that you build the binaries and libpng is installed:
https://github.com/nodejs/node-gyp#on-macos
